base/profile.jl

```diff
@@ -121,11 +121,50 @@ end
 
 function getdict(data::Vector{UInt})
     uip = unique(data)
-    Dict{UInt, StackFrame}(map(uip) do ip
-        lk = lookup(ip)
-        # TODO handle inlined frames
-        ip => first(lk)
-    end)
+    Dict{UInt, Vector{StackFrame}}(ip=>lookup(ip) for ip in uip)
+end
+
+"""
+    newdata, newdict = flatten(btdata, lidict)
```

Comment:
`flatten(btdata, lidict) -> (newdata, newdict)` would be more consistent with other docstrings